node-graph/gstd/src/any.rs

```diff
@@ -31,7 +31,8 @@ where
 {
 	type Output = Any<'n>;
 	fn eval(self, input: Any<'n>) -> Self::Output {
-		let input: Box<I> = dyn_any::downcast(input).expect("DynAnyNode Input");
+		let node = core::any::type_name::<N>();
+		let input: Box<I> = dyn_any::downcast(input).expect(format!("DynAnyNode Input \n{node}").as_str());
```

Comment:
Logging the node here is a good debugging improvement.

Reply:
Yeah, but I should change the formatting + truncate long node types, e.g. `ComposeNode<…>`